full_fugu.mk

```diff
@@ -14,10 +14,24 @@
 # limitations under the License.
 #
 $(call inherit-product, device/asus/fugu/device.mk)
-$(call inherit-product, device/google/atv/products/atv_base.mk)
+#$(call inherit-product, device/google/atv/products/atv_base.mk)
```

Comment:
Can't we just make sure PRODUCT_PACKAGES is controlled? I fear we might miss other bits.